streampipes-extensions/streampipes-processors-geo-jvm/pom.xml

```diff
@@ -63,7 +63,17 @@
         <dependency>
             <groupId>org.locationtech.jts</groupId>
             <artifactId>jts-core</artifactId>
-            <version>1.16.1</version>
+            <version>1.19.0</version>
```

Comment:
Hi @flomickl can you please add the dependencies to the <dependencyManagement> tag in the parent pom. 
Then you don't need the version number in this pom. 
In the parent pom you can also create a variable like it is done for other dependencies.
If you have any questions, just let me know.

Reply:
@tenthe You mean here?
https://github.com/apache/streampipes/blob/dev/pom.xml ?

What is the difference between these two pom files?
The change should look something like:

```
...
<jts.version>1.19.0</jts.version>
...

<dependency>
  <groupId>org.locationtech.jts<</groupId>
  <artifactId>jts-core</artifactId>
  <version>${jts.version}</version>
</dependency>
```

Should I do it for these entries?

- jts
- postgresql  -> pretty sure that this entry already exist sin the parent pom
- sis
- com.squareup.okhttp

I did not add 

```
        <dependency>
            <groupId>com.squareup.okhttp3</groupId>
            <artifactId>okhttp</artifactId>
            <version>3.13.1</version>
        </dependency>
```


The entry in the geo.jvm pom should also be changed to following entry?

```
<dependency>
  <groupId>org.locationtech.jts<</groupId>
  <artifactId>jts-core</artifactId>
  <version>${jts.version}</version>
</dependency>
```

Reply:
Hey @flomickl,
the parent pom ensures that we use the same version of a dependency within the whole project.
Therefore, the dependency must be added to the `<dependencyManagement>` tag with the version in the parent pom.xml.
To use this dependency you can add it the to the pom.xml in the module that requires the dependency to the `<dependencies>` tag. The version should only be defined in the parent pom.xml and not in the module pom.xml.

The only exception are the StreamPipes dependencies.

Does this help?

Reply:
@tenthe I updated the pom files.
I also run manual tests again. Everything as expected.
I left this inside, but sure that this could be removed in another ticket. The version is also outdated in both pom files.

```
        <dependency>
            <groupId>com.squareup.okhttp3</groupId>
            <artifactId>okhttp</artifactId>
            <version>3.13.1</version>
        </dependency>
```

Reply:
The parent pom would also need new code format process in an extra commit!
Did not want to add this here as well